Mage.Server.Plugins/Mage.Game.BrawlFreeForAll/src/mage/game/BrawlFreeForAll.java

```diff
@@ -26,9 +26,9 @@ public BrawlFreeForAll(final BrawlFreeForAll game) {
     }
 
     @Override
-    protected void init(UUID choosingPlayerId) {
+    protected void init(UUID choosingPlayerId, boolean resetWatcher) {
         startingPlayerSkipsDraw = false;
-        super.init(choosingPlayerId); 
+        super.init(choosingPlayerId, resetWatcher);
```

Comment:
Why you need to skip watchers reset on game start?

Reply:
Calling execute in the middle of the turn resets the watchers, as game.start() is called. I'll make a separate PR for that, with instead of a added boolean I'll add a setWatcherResetAtStart in GameImpl, and call that directly.

Reply:
No needs in that. Multiple executes in unit test are not normal use case and it has another limits/bugs. Use checkXXX commands for real time checks -- this is the preferred option.

Reply:
Ok thanks, I totally missed we add those checkXXX to check in the middle of execution.